skins/kama/dialog.css

```
@@ -352,6 +352,11 @@ div.cke_disabled .cke_dialog_ui_labeled_content *
 	cursor : default;
 }
 
+/* (#3433) */
+.cke_dialog_ui_labeled_required{
+	color: red;
+}
```

Let's stick to the file styling.
```suggestion
.cke_dialog_ui_labeled_required
{
	color: red;
}
```